src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -166,10 +156,10 @@ private void configMotors() {
         hoodMotor.setInverted(true);
         hoodMotor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kForward, true);
         hoodMotor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, true);
+        // Current hood setup plan start hood at 0, below MIN_HOOD_ANGLE
         hoodMotor.setSoftLimit(CANSparkMax.SoftLimitDirection.kForward,
                 (float) (MAX_HOOD_ANGLE / HOOD_REVS_TO_DEGREES));
-        hoodMotor.setSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, 0); // Current hood setup plan starts hood at 0,
-                                                                            // below MIN_HOOD_ANGLE
```

Comment:
The comment is specific to setting the reverse soft limit.

Reply:
How did i move that comment to the wrong line...